dockerfiles/Dockerfile-snapp-test-txn

```diff
@@ -0,0 +1,40 @@
+ARG deb_codename=stretch
+FROM debian:${deb_codename}-slim
+
+# Run with `docker build --build-arg deb_version=<version>`
+ARG deb_version
+ARG deb_codename=stretch
+ARG deb_release=alpha
+
+ENV DEBIAN_FRONTEND noninteractive
+RUN echo "Building image with version $deb_codename $deb_release $deb_version"
+
+COPY scripts/archive-entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
+
+# Dependencies
+RUN apt-get -y update \
+    && apt-get -y upgrade \
+    && apt-get -y install \
+        procps \
+        curl \
+        jq \
+        libjemalloc-dev \
+        dumb-init \
+        libssl1.1 \
+        libpq-dev \
+        apt-transport-https \
+        ca-certificates \
+        dnsutils \
+	apt-utils \
+        man \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /healthcheck && curl https://raw.githubusercontent.com/MinaProtocol/mina/develop/dockerfiles/scripts/healthcheck-utilities.sh -o /healthcheck/utilities.sh
```

Comment:
Should the `develop` branch here be a variable?

Reply:
consulting with Andrew on that one

Reply:
removed that line